llvm/lib/CodeGen/TargetLoweringBase.cpp

```diff
@@ -407,6 +407,11 @@ RTLIB::Libcall RTLIB::getFSINCOS(EVT RetVT) {
                       SINCOS_PPCF128);
 }
 
+RTLIB::Libcall RTLIB::getSINCOSPI(EVT RetVT) {
```

Comment:
The naming looks inconsistent here albeit the new code looks more correct.  Any chance you can follow up and rename `getFSINCOS` to `getSINCOS`?

Reply:
Will do :+1: